Added a Question in ML section ("Classification not Regression")

sanidhyas3s opened the pull request:
- added a descriptive question and solution to the yml file (ML).

sanidhyas3s committed:
Added a Question ("Classification not Regression")

- added a descriptive question and solution to the yml file (ML).

gargdoppler reviewed (changes requested):
- Fix indent for the key `title` in `questions/ml.yml`.
- Not sure if the `eda` tag is valid.
- Add tags like `cv`, `ann`.

sanidhyas3s committed:
made the required changes in the question

- fixed indent in title and changed tags as asked

sanidhyas3s commented:
Made the changes needed.

gargdoppler reviewed (dismissed):
Try to add questions at the end of the file.

sanidhyas3s committed:
Shifted the question to the end of the file

sanidhyas3s commented:
> Try to add questions at the end of the file.

Done!

sanidhyas3s committed:
Merge branch 'dsgiitr:main' into patch-2

sanidhyas3s committed:
Updated the formatting of the question added.

Updated the formatting of the question added after merging with the updated state of old questions.

sanidhyas3s commented:
@Garg-Doppler Updated to the new format, you can review again.
